src/main/java/github/grace000/erversay/RequestRouter.java

```diff
@@ -0,0 +1,22 @@
+package github.grace000.erversay;
+
+import java.util.HashMap;
+
+import static github.grace000.erversay.Constants.*;
+
+public class RequestRouter {
+    private HashMap<String, String> routeTable = new HashMap<>();
+
+    public HashMap route(String[] requestLineArray) {
```

Comment:
It seems inconvenient and error prone to use a `String[]` here. Why not a `Request` object with explicit fields, like `uri`?

Reply:
That's what I was working towards. I now have a Request object instead :)